### PR TITLE
adds stephanie talk

### PR DESCRIPTION
**Pull request by fboxwala**

### CODE REVIEW

File: content/W2016/W16-event-stephanie-talk.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Title: Out of Town Speaker: Stephanie Morillo
+Date: 2016-02-15
```
Comment (ehashman) [inline content/W2016/W16-event-stephanie-talk.md]: Need to update this.

File: content/W2016/W16-event-stephanie-talk.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Slug: stephanie-morillo-talk
+Author: Fatema Boxwala
+Summary: Come out to hear Stephanie Morillo talk about her experiences in the tech world!
+
+WiCS Undergrad is proud to invite Stephanie Morillo as our out-of-town speaker
+this term! Stephanie will be joining us on March 21st to give a talk entitled:
+“Circuit Breaker: How My Love For Writing Brought Me To Tech.”
+
+Abstract: The road to a successful career in tech wasn’t forged back in 
+university; it was forged in my mid-twenties when I was unemployed and learned to program on a
```
Comment (ehashman) [inline content/W2016/W16-event-stephanie-talk.md]: This line needs wrapping.

**Comment by ehashman** [comment]
Other than those small things, LGTM.

**Comment by ehashman** [comment]
Still waiting on updates.

**Comment by fboxwala** [comment]
Sorry, midterms etc. Should be good now.

**Pull request merged**